Network/centurylink-cloud-public-ip-listing.md

```diff
@@ -0,0 +1,124 @@
+{{{ "title": "CenturyLink Cloud Public IP Listing",
+"date": "07-06-2016",
+"author": [],
+"attachments": [],
+"contentIsHTML": false
+}}}
+
+
+CenturyLink Cloud customers may wish to add a public IP to specific [virtual machine](/servers/) or [Bare Metal](/bare-metal/) server in their cloud environment to deliver services. Public IPs are delivered using a 1 to 1 NAT model.
+
+### General Notes & Best Practices
+
+* All public IPs deployed on the platform have [hairpinning](/knowledge-base/network/hairpin-nats/) enabled.
+* In its current iteration setting a source IP filter will secure all public ports, single ports or port ranges specified by the customer. Customers can leverage OS based [firewall services](/cloud-firewall/) if they wish to secure public services in a more granular fashion.
+* Customers are encouraged to leverage the source IP filter unless delivering completely open public Internet services to their user community.
+* Customers should avoid opening RDP or SSH to their virtual machines to the public Internet. As such the following are recommended access methods.
+
+ 1. Use the free OpenVPN client included in every CenturyLink Cloud Account. Refer to [How To Configure Client    VPN](/knowledge-base/network/how-to-configure-client-vpn/). This is the ideal solution for individuals who are mobile and not in fixed office or data center locations.
+ 2. Build an IPSEC VPN Tunnel from a remote office or data center location. Refer to [Creating a Self-Service IPSEC Site-to-Site VPN Tunnel](/knowledge-base/network/creating-a-self-service-ipsec-site-to-site-vpn-tunnel/). IPSEC VPN tunnels are best for remote access to Cloud Virtual Machines when administrators are in centralized offices or data centers.
```

Comment:
Use relative path `../Network/creating-a-self-service-ipsec-site-to-site-vpn-tunnel.md`